linkerd/app/gateway/src/lib.rs

```diff
@@ -2,6 +2,7 @@
 #![forbid(unsafe_code)]
 
 use linkerd_app_core::{
+    addr::NameMatch,
```

Comment:
Can we preserve the reexports so we don't have to change all of these in this PR? sorry.